native/SampleApps/AuthFlowTester/AuthFlowTester/Views/UserCredentialsView.swift

```diff
@@ -28,103 +28,188 @@
 import SwiftUI
 import SalesforceSDKCore
 
+// MARK: - Labels Constants
+public struct CredentialsLabels {
```

Comment:
Pulled the labels into a struct.
- they are used also when exporting details as JSON.
- they are also used in UI tests.